docs/BR.md

```diff
@@ -199,7 +199,7 @@ Certification Authority (CA) is defined in [Section 1.6](#16-definitions-and-acr
 
 ### 1.3.2 Registration Authorities
 
-With the exception of [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address), the CA MAY delegate the performance of all, or any part, of [Section 3.2](#32-initial-identity-validation) requirements to a Delegated Third Party, provided that the process as a whole fulfills all of the requirements of [Section 3.2](#32-initial-identity-validation).
```

Comment:
Although this seems to have been implicitly required, if we are to make it explicit, we probably will need some sort of effective date (something like ~30 days after the IPR period clears).

Reply:
I've added an effective date of 2024-05-15, expected to be between 1 and 2 months after the IPR period and nicely aligned with the "middle of odd months" idea.